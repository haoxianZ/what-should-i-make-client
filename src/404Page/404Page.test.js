import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import NotFound from './404Page';

describe(`404 Page component`, () => {
  it('renders the complete form', () => {
    const wrapper = shallow(<NotFound />);
    expect(toJson(wrapper)).toMatchSnapshot();
  })
}) 