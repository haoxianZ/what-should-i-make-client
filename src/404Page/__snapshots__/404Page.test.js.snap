// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`404 Page component renders the complete form 1`] = `
<div
  className="notPage"
>
  <Link
    style={
      Object {
        "textDecoration": "none",
      }
    }
    to="/home-page"
  >
    <header>
      What Should I Make?
    </header>
  </Link>
  <h3
    className="sthWrong"
  >
    Oooops! Something went wrong.
    <br />
  </h3>
</div>
`;
